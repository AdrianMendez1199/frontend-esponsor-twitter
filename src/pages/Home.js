import React from 'react';
import Card from '../components/Card';
import Navbar from '../components/Navbar';
import { useCtxUser } from '../userContext';

const Home = () => {
  const [user] = useCtxUser();
  return (
    <div>
      <Navbar />
      <main>
        {user && <Card username={user.name} /> }
      </main>
    </div>
  );
};

export default Home;
