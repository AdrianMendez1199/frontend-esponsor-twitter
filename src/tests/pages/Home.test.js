import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import Home from '../../pages/Home';

describe('testing <Home />', () => {
  const wrapper = shallow(<Home />);


  test('should match with Home snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should find div home container', () => {
    expect(wrapper.contains(<div className="home-container" />));  
  })
});
