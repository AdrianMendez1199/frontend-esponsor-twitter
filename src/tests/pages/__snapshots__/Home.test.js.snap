// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing <Home /> should match with Home snapshot 1`] = `
<div
  className="home-container"
>
  <Navbar />
  <div
    className="home-content"
  />
  <div
    className="top"
  />
</div>
`;
